--- a/src/plugins/CMakeLists.txt        2022-02-24 15:09:32.984235101 +0100
+++ b/src/plugins/CMakeLists.txt        2022-02-24 15:08:41.663698835 +0100
@@ -19,11 +19,18 @@
 ##############################################################################
 # find and add all plugin subdirs
 ##############################################################################
+set(ALLOWED_PLUGINS "dpdk" "upf")
+
 FILE(GLOB files RELATIVE
   ${CMAKE_CURRENT_SOURCE_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}/*/CMakeLists.txt
 )
 foreach (f ${files})
   get_filename_component(dir ${f} DIRECTORY)
-  add_subdirectory(${dir})
+  IF(${dir} IN_LIST ALLOWED_PLUGINS)
+          message("Adding sudirectory -> " ${dir})
+          add_subdirectory(${dir})
+  ENDIF()
+  #add_subdirectory(${dir})
 endforeach()
+

